proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -60,6 +59,35 @@ message MsgCommunityPoolSpend {
 
 // MsgCommunityPoolSpendResponse defines the response to executing a
 // MsgCommunityPoolSpend message.
-//
-// Since: cosmos-sdk 0.50
-message MsgCommunityPoolSpendResponse {}
\ No newline at end of file
+message MsgCommunityPoolSpendResponse {}
+
+// MsgSubmitBudgetProposal defines budget proposal type.
+message MsgSubmitBudgetProposal {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string                   authority          = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string                   recipient_address  = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  cosmos.base.v1beta1.Coin total_budget       = 3;
+  int64                    start_time         = 4;
+  int64                    remaining_tranches = 5;
```

Comment:
nit: probably should be just `tranches` remaining does not make too much sense when you submit the proposal